tianshou/highlevel/experiment.py

```diff
@@ -249,7 +283,7 @@ def run(
             enabled=use_persistence and self.config.log_file_enabled,
         ):
             # log initial information
-            log.info(f"Running experiment (name='{run_name}'):\n{self.pprints()}")
+            log.info(f"Running experiment (name='{self.name}'):\n{self.pprints()}")
```

Comment:
This, too, is a regression.

Reply:
Some copy-paste changes from our internal repo that wasn't synced with upstream snuck in, gonna undo them

Reply:
I restored backwards compat in the `run` method. 

Here I don't think it's a regression, run will always pass `override_experiment_name` to this method, which will then set it to the `name` attr.

Reply:
I think you're misinterpeting the diff, this is no longer the `run` method but the `create_experiment_world` method. The param cannot be called `run_name`, and if I'm not mistaken the functionality of  `run` has not changed. If I'm wrong, let's have a short call to clear this up.

Reply:
Ah, wait, I think I misread the diff or copied something wrongly from there. Now I see what you mean. 

It can't be called `run_name` here since this is not a run, but I will change the modification of the name attr that I re-introduced

Reply:
Done